src/main/java/by/itechart/api/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package by.itechart.api.service.impl;
+
+import by.itechart.api.entity.User;
+import by.itechart.api.repository.UserRepository;
+import by.itechart.api.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class UserServiceImpl implements UserService {
+
+    private final UserRepository userRepository;
+
+    @Override
+    public User create(User user) {
+        return userRepository.save(user);
+    }
+
+    @Override
+    public User update(User user) {
+        return userRepository.saveAndFlush(user);
+    }
+
+    @Override
+    public void delete(Long id) {
+        userRepository.deleteById(id);
+    }
+
+    @Override
+    public List<User> findAll() {
+        return userRepository.findAll();
```

Comment:
Taking into account the `logical/soft deletion` mentioned above, I think we can leave this method as it is.
Later we will add some methods for searching